### DISCUSSION TITLE
catboost_spark for WSL

### DISCUSSION THREAD

**Discussion by ArijitSinghEDA** [catboost/catboost#2656] [Q&A] [answered]
I have followed the instructions given on the official website to install catboost for spark. The moment I try training a model, I get `ModuleNotFoundError: No module named 'catboost_spark'`. I am running this in WSL. I know Windows is currently not supported, does it extend to WSL as well, even if it provides a Linux environment?

**Comment by andrey-khropov** [comment, marked as answer]
You are right, this limitation does not extend to WSL. CatBoost for Apache Spark should work in WSL (although I haven't checked it personally). If you install according [to the instructions](https://catboost.ai/en/docs/concepts/spark-installation) and have problems, please, create a separate GitHub issue.
